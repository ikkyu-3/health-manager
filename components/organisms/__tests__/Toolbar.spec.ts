// import Vuex from 'vuex'
// import Vuetify from 'vuetify'
// import { mount, createLocalVue } from '@vue/test-utils'
// import Toolbar from '@/components/organisms/Toolbar.vue'

// const localVue = createLocalVue()
// localVue.use(Vuex)

// const options = {
//   localVue,
//   vuetify: new Vuetify()
// }

// jest.mock('@/store', () => ({
//   userStore: () => new Vuex.Store({ state: { title: 'Workouts' } })
// }))

// describe('organisms/Toolbar.vue', () => {
//   it('', () => {})
// })
