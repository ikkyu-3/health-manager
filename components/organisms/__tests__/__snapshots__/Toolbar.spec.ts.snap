// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`organisms/Toolbar.vue snaphot 1`] = `"<div><header class=\\"v-sheet v-sheet--tile theme--dark v-toolbar\\" style=\\"height: 64px; background-color: rgb(25, 118, 210); border-color: #1976d2;\\"><div class=\\"v-toolbar__content\\" style=\\"height: 64px;\\"><div class=\\"v-toolbar__title\\">Workouts</div></div></header></div>"`;
